src/main/scala/ctrl.scala

```diff
@@ -79,23 +79,29 @@ class CtrlModule(val W: Int, val S: Int)(implicit p: Parameters) extends Module(
 
   val buffer_sram = p(BufferSram)
   //SRAM Buffer
-  val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
+  //val buffer_mem = Mem(UInt(width = W), round_size_words, seqRead = true)
```

Comment:
We don't need to keep around the commented out version. Git will handle that for us.